catalog/ui/search-ui/standard/src/main/webapp/js/module/Tasks.module.js

```diff
@@ -29,7 +29,7 @@ define(['application',
             TaskModule.addInitializer(function() {
 
                 this.subscription = Cometd.Comet.subscribe("/ddf/activities/**", function(resp) {
-                    var task = new Task.Model(resp, {validate: true, parse: true});
+                    var task = new Task.Model(resp.data, {validate: true, parse: true});
```

Comment:
parse always gets called by backbone when creating a new object. That is just the point that you'd do anything additional to the data before the object is created.